htmlgenerator/base.py

```diff
@@ -77,7 +77,9 @@ def render(self, context: dict) -> typing.Generator[str, None, None]:
 
     def filter(
         self,
-        filter_func: typing.Callable[[BaseElement, typing.Iterable[BaseElement]], bool],
+        filter_func: typing.Callable[
+            [BaseElement, typing.Tuple[BaseElement, ...]], bool
```

Comment:
just for my education: what does the ellipsis mean?

Reply:
They describe an arbitrary length of types of the same element. A tuple-type has a fixed length but in order to express that we accept a tuple of any lenght we can use the three dots. I think the dots can also be used with ```typing.Callable```.